code.cpp

```diff
@@ -0,0 +1,116 @@
+#include <array>
+#include <iostream>
+#include <list>
+#include <unistd.h>
+
+class Table{
+    public:
+        static const int width = 32;
+        static const int height = 11;
+        static const int table_size =  width * height;
+};
+
+
+std::array<int,Table::table_size> step(std::array<int,Table::table_size>& arr) {
+    std::array<int, Table::table_size> res; 
+    for (int i = 0; i < Table::height; i++) {
+        for (int j = 0; j < Table::width; j++) {
+            int count = 0;
+            if (i > 0 && j > 0 && arr[Table::width * (i - 1) + (j - 1)] == 1) {
+                count++;
```

Comment:
You can do `count += (i > 0 && j > 0 && arr[Table::width * (i - 1) + (j - 1)] == 1)`